webapi/src/main/scala/org/knora/webapi/slice/common/domain/InternalIri.scala

```diff
@@ -5,12 +5,20 @@
 
 package org.knora.webapi.slice.common.domain
 
+import scala.util.Try
+
 import org.knora.webapi.IRI
 import org.knora.webapi.messages.SmartIri
+import org.knora.webapi.messages.StringFormatter
 import org.knora.webapi.slice.common.KnoraIris.KnoraIri
 
 final case class InternalIri(value: IRI)
 object InternalIri {
   def from(knoraIri: KnoraIri): InternalIri = from(knoraIri.smartIri)
   def from(smartIri: SmartIri): InternalIri = InternalIri(smartIri.toInternalSchema.toIri)
+  def from(iri: IRI)(implicit sf: StringFormatter): Either[Throwable, InternalIri] =
```

Comment:
I would recommend to use the `IriConverter` instead of adding this from method. 

```org.knora.webapi.slice.common.service.IriConverter#asInternalIri```

I'd like to keep the use of implicitly passing the StringFormatter to a minimum.